nss-day-cohort-60/national-parks-client
# add styles

# Description
Gave the landing page a dynamic background behind the carousel. Added conditional rendering to the PhotoCarousel module based on the URL so it's still reusable in different places. Made the text on the landing page hierarchical. Made the images of the parks the same size in the parks list. Changed the title in the HTML to Park explorer. Changed the big title on the landing page to an h1 element. Added a text rotating animation to the title. Made the Navbar sticky.

# Steps to test

1. Fetch the <hp-ad-styles> branch and switch to it (Not to be confused with hp-add-styles, as i made a typo)
2. Start/restart debugger
3. Go to http://localhost:3000/home
4. Do you see it?
